.ci/install_ovmf_x86_64.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Copyright 2022 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+cidir=$(dirname "$0")
+source "${cidir}/lib.sh"
+
+main() {
+	local buildscript="${katacontainers_repo_dir}/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh"
+
+	# Just in case the kata-containers repo is not cloned yet.
+	clone_katacontainers_repo
+
+	pushd $katacontainers_repo_dir
+	sudo -E PATH=$PATH bash ${buildscript} --build=cc-x86_64-ovmf
+	sudo tar -xvJpf build/kata-static-cc-x86_64-ovmf.tar.xz -C /
+	sudo ln -sf /opt/confidential-containers/share/ovmf /usr/share/ovmf
```

Comment:
Why are we doing this?

Neither the payload image, or the operator, will be doing such a link.  We need to ensure that Kata Containers is looking at the correct place instead of linking something to `/usr/share/...`

Reply:
@fidencio I think this is one step in the same direction. I am removing multiple generic functions from SEV specific files to a common place. This will help if we decide to standardize all the TEE tests. Since all tests can use this same common code.

Reply:
Thanks for this. Fixed the symlink for this and for AMDSEV firmware as well.